authority/authority.go

```diff
@@ -71,6 +77,22 @@ func New(config *Config) (*Authority, error) {
 	return a, nil
 }
 
+// Initialize database with all necessary tables.
+func (a *Authority) initDB() error {
+	var err error
+	if a.db, err = db.New(a.config.DB); err != nil {
+		return err
+	}
+	tables := [][]byte{revokedCertsTable}
+	for _, b := range tables {
+		if err = a.db.CreateTable(revokedCertsTable); err != nil {
+			return errors.Wrapf(err, "error creating table %s",
+				string(b))
+		}
+	}
+	return nil
+}
```

Comment:
I think is better to move all the db logic to the new package `github.com/smallstep/certificates/db`